nuxt.config.ts

```diff
@@ -41,6 +39,12 @@ export default defineNuxtConfig({
     trailingSlash: true,
   },
 
+  ogImage: {
```

Comment:
`@nuxt/image` installs sharp but doesn't resolve the binary properly for me (ignore this)